### DISCUSSION TITLE
Frozen demo boot

### DISCUSSION THREAD

**Discussion by GlamrockFreddy127** [ipxe/ipxe#718] [Q&A] [answered]
I am attempting to boot the iPXE demo and it freezes about 5% through. Do you guys know how to fix it?

**Comment by mcb30** [comment]
Please provide a screenshot so that we can see what is happening.

**Comment by GlamrockFreddy127** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/98360095/186269864-42f37b3e-e743-43ce-a76b-2d25e17c23c4.png)

**Reply by stappersg** [reply]
The original report says _it freezes about 5% through_, the screenshot says 16%.My guess is that the _"freeze"_ is just _impatience_.

GlamrockFreddy127: No hard feelings, please do provide further information about what you experience as discomfort.

**Reply by GlamrockFreddy127** [reply]
It is actually frozen,it has not went up since about 10 minutes ago.

Please,can you try to find a way to fix it?

**Reply by NiKiZe** [reply]
Does it always hang on the same %?
Can you download the kernel/file from a browser on your host without issues?

Likely causes for your issue is: bad network/internet or broken config on your VM host.
None of these are things that we can fix for you :/
With that said we can try to help you find the issue, and maybe that will lead you to find a fix.

**Reply by GlamrockFreddy127** [reply]
It doesn't let me download it,it always displays text.
Any idea how to download it?

**Reply by mcb30** [reply]
At the point of freezing, does Ctrl-C allow you to cancel the download and get back to a command prompt?

Please show us the output of `ifstat` before attempting the download, and after using Ctrl-C to cancel the frozen download.

**Reply by GlamrockFreddy127** [reply]
After running ifstat,this is what i got...
![image](https://user-images.githubusercontent.com/98360095/186286047-60902a40-b460-49ce-b0fd-08ee3a7bee9b.png)

**Reply by GlamrockFreddy127** [reply]
I don't know what to do at this point. Tell me if you've found a fix!

**Reply by NiKiZe** [reply]
Please follow the instructions fully, and provide the requested information as requested.

When you ignore parts of the requested information it gets hard (if not impossible) to help you.

**Reply by GlamrockFreddy127** [reply]
Oh,and if you want to know what config my host has,here it is:
Host Model:HP Pavilion 15-cd040wm
Host OS:Windows 11 Home
VMware Version:16.2.4 (Licensed)
Other Hypervisors:VirtualBox, Hyper-V, QEMU

**Reply by GlamrockFreddy127** [reply]
I even tried it in VirtualBox.
![image](https://user-images.githubusercontent.com/98360095/186444490-db5a1384-9985-4f93-8ad6-60bd3d6e2968.png)

**Reply by GlamrockFreddy127** [reply]
Well,if all else fails,i can just try Hyper-V or QEMU...Right?

**Reply by stappersg** [reply]
Advice: 
* Take a walk  outdoors.
* Reread what was discussed.
* Follow-up on the requests,  (especial on your on request)

**Reply by GlamrockFreddy127** [reply]
QEMU didn't work...What's next,Hyper-V freaking out?
![image](https://user-images.githubusercontent.com/98360095/186446141-522157ff-2ec9-4a06-a757-3fcf09b7de7f.png)

**Reply by GlamrockFreddy127** [reply]
Hyper-V actually worked!
![image](https://user-images.githubusercontent.com/98360095/186450795-dc4e6ca3-ceff-427e-bafe-fcc156f0cc0d.png)

**Reply by NiKiZe** [reply]
The suspicion of network issues still stands, and so does "download the files in your browser"

One workaround for you might be to set up a local http service with the files.


Please understand that issues with your network or internet connection is NOTHING that anyone else really can help you with.
